lib/utilities/dateChecker.ts

```diff
@@ -0,0 +1,12 @@
+import moment from 'moment'
+/**
+ * Formats a timestamp into a readable date format regardless of seconds or milliseconds
+ *
+ */
+const dateChecker = (date: number) => {
+  const isMilliSeconds = date && date.toString().length >= 13
```

Comment:
Converting to string and checking length is a much more complicated operation than just doing a greater than on a number.